cabal-install/Distribution/Client/Setup.hs

```diff
@@ -1166,12 +1166,12 @@ outdatedCommand = CommandUI {
     optionVerbosity outdatedVerbosity
       (\v flags -> flags { outdatedVerbosity = v })
 
-    ,option [] ["freeze-file"]
+    ,option [] ["freeze-file", "v1-freeze-file"]
      "Act on the freeze file"
      outdatedFreezeFile (\v flags -> flags { outdatedFreezeFile = v })
      trueArg
 
-    ,option [] ["new-freeze-file"]
+    ,option [] ["new-freeze-file", "v2-freeze-file"]
```

Comment:
I'd rather make freeze files state their format than rely on user to remember it right.

Reply:
Freeze file format is actually the same for v1 and v2, but the file names are different.